Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM python:3.7-stretch
+
+# todo: should be user home dir
+ENV HOME_DIR=/root
```

Comment:
I tried using `$HOME` for this but there are issues with that variable in Dockerfiles, according to...various discussions when Googling.